.travis.yml

```diff
@@ -22,14 +22,17 @@ matrix:
     - env: EMBER_TRY_SCENARIO=ember-canary
 
 before_install:
-  - export PATH=/usr/local/phantomjs-2.0.0/bin:$PATH
-  - "npm config set spin false"
-  - "npm install -g npm@^2"
+  - npm config set spin false
+  - npm install -g bower
+  - bower --version
+  - npm install -g phantomjs-prebuilt
+  - node_modules/phantomjs-prebuilt/bin/phantomjs --version
```

Comment:
@cyril-sf, could you fix this to make Travis CI pass? 
Right now it reports: [`node_modules/phantomjs-prebuilt/bin/phantomjs: No such file or directory
`](https://travis-ci.org/alsemyonov/ember-faye/jobs/226135674#L297)